micromasters/settings.py

```diff
@@ -421,3 +422,12 @@ def get_var(name, default):
 
 # django-role-permissions
 ROLEPERMISSIONS_MODULE = 'roles.roles'
+
+CELERYBEAT_SCHEDULE = {
+    'batch-update-user-data-every-6-hrs': {
+        'task': 'dashboard.tasks.batch_update_user_data',
+        'schedule': crontab(minute=0, hour='*/6')
+    },
+}
+
+CELERY_TIMEZONE = 'UTC'
```

Comment:
then it wont work because it should be in settings
https://github.com/mitodl/micromasters/blob/master/micromasters/celery.py#L21